### DISCUSSION TITLE
Spring Boot we have one Camel Router & JMS Queue

### DISCUSSION THREAD

**Discussion by jkumar0021** [javax4u/interview-questions#95] [Q&A]
Q. In Spring Boot we have one Camel Router which is running every 30 min, While Running the router how many thread will run concurrently to complete the job.
Q. In Spring Boot, we have JMS Consumer to consume the JMS from Queue message, suddenly one lack message come in the queue, how many thread will concurrently start to consume the message?
Q. After consuming the message, are you deleting the msg from queue  
Q. How did you implement DL Queue concept in ActiveMQ?

**Comment by javax4u** [comment]
Q. Connection between producer and consumer is single-thread or multi-thread ?
